docs/quick.md

```diff
@@ -14,61 +14,78 @@ meets the following prerequisites:
     NAME                                    READY   STATUS    RESTARTS   AGE
     openebs-ndm-f4kzc                       1/1     Running   0          37s
     openebs-ndm-operator-796f98fdd7-kvmpn   1/1     Running   0          37s
     openebs-ndm-t65b5                       1/1     Running   0          37s
     openebs-ndm-xztkj                       1/1     Running   0          37s
-
     ```
     Check that blockdevices are created:
+
     ```bash
-    kubectl get bd -n openebs
-    ```
+    $ kubectl get bd -n openebs
+    NAME                                           NODENAME           SIZE          CLAIMSTATE   STATUS   AGE
+    blockdevice-01afcdbe3a9c9e3b281c7133b2af1b68   worker3            21474836480   Unclaimed    Active   2m10s
+    blockdevice-10ad9f484c299597ed1e126d7b857967   worker1            21474836480   Unclaimed    Active   2m17s
+    blockdevice-3ec130dc1aa932eb4c5af1db4d73ea1b   worker2            21474836480   Unclaimed    Active   2m12s
     ```
-     NAME                                           NODENAME           SIZE          CLAIMSTATE   STATUS   AGE
-    blockdevice-01afcdbe3a9c9e3b281c7133b2af1b68    worker3            21474836480   Unclaimed    Active   2m10s
-    blockdevice-10ad9f484c299597ed1e126d7b857967    worker1            21474836480   Unclaimed    Active   2m17s
-    blockdevice-3ec130dc1aa932eb4c5af1db4d73ea1b    worker2            21474836480   Unclaimed    Active   2m12s
 
-    ```
     NOTE: 
     1. It can take little while for blockdevices to appear when the application is warming up.
     2. For a blockdevice to appear, you must have disks attached to node.
 
 5.  Install cStor CRDs.
     ```bash
-    kubectl create -f deploy/crds
+    $ kubectl create -f deploy/crds
     ```
 
 6.  Install cStor operators.
     ```bash
-    kubectl create -f deploy/cstor-operator.yaml
+    $ kubectl create -f deploy/cstor-operator.yaml
     ```
     Check that cspc-operator, cvc-operator and admission server pod has came up.
-   
+
+    ```bash
+    $ kubectl get pods -n openebs
+    NAME                                              READY   STATUS    RESTARTS   AGE
+    cspc-operator-874cdcb6b-t4zcs                     1/1     Running   0          25s
+    cvc-operator-fbcf99548-swlbg                      1/1     Running   0          25s
+    openebs-cstor-admission-server-7c89777f8c-bbgwp   1/1     Running   0          25s
+    openebs-ndm-f4kzc                                 1/1     Running   0          6m5s
+    openebs-ndm-operator-796f98fdd7-kvmpn             1/1     Running   1          6m5s
+    openebs-ndm-t65b5                                 1/1     Running   0          6m5s
+    openebs-ndm-xztkj                                 1/1     Running   0          6m5s
+    ```
+
+7. Install CStor CSI driver.
+
+   ```bash
+    $ kubectl create -f deploy/csi-operator.yaml
+   ```
+    Check that cstor-csi-controller and cstor-csi-node pods has came up.
+
     ```bash
+    $ kubectl get pods -n openebs
```

Comment:
To verify cStor-CSI related pods status, Do they have to verify in `kube-system` namespace?

Reply:
yes we have to change to kube-system for this release ,and update back to openebs later